staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/metrics/metrics.go

```diff
@@ -120,3 +151,11 @@ func (m *ValidatingAdmissionPolicyMetrics) ObserveWarn(ctx context.Context, elap
 	m.policyCheck.WithContext(ctx).WithLabelValues(policy, binding, string(errorType), "warn").Inc()
 	m.policyLatency.WithContext(ctx).WithLabelValues(policy, binding, string(errorType), "warn").Observe(elapsed.Seconds())
 }
+
+// ObserveActiveBindings observes the count of active validation admission policy bindings.
+func (m *ValidatingAdmissionPolicyMetrics) ObserveActiveBindings(policyCount, allowCount, warnCount, denyCount int) {
```

Comment:
active policy number should equal to active binding number. As long as we add enforcement action info into the metric and user should be able to filter directly?

Reply:
There could be multiple bindings per policy. I was intending to distinguish between the policy resources and the binding resources, however I'm not wedded to this so can change if that's not what you would expect.

Reply:
Ohh I thought we wanna count policy-binding pair but counting numbers for active policy/bindings separately also makes sense :)